modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -89,6 +89,11 @@ export default {
     },
     manuallyPublished() {
       return this.moduleOptions.localized && !this.moduleOptions.autopublish;
+    },
+    showLocalePicker() {
+      return Object.keys(window.apos.i18n.locales).length > 1 &&
+        this.moduleOptions.localized !== false &&
+        !this.modalData.hasContextLocale;
```

Comment:
This computed also exist in `AposDocEditor` that use the same mixin, you should remove it from there.